core/src/test/scala/integration/kafka/api/ConsumerTest.scala

```diff
@@ -254,6 +255,17 @@ class ConsumerTest extends IntegrationTestHarness with Logging {
     }
   }
 
+  def testPartitionPauseAndResume() {
```

Comment:
Could we add another unit test for the behavior that the pause state should be cleared  across consumer rebalances?